config.xml

```diff
@@ -39,6 +41,7 @@
         <package name="hashmyfiles.vm"/>
         <package name="hollowshunter.vm"/>
         <package name="hxd.vm"/>
+        <package name="ida.diaphora.vm"/>
```

Comment:
I like this tool and would like it in the VM. But it just copies the file into the VM instead of installing it as plugin. I thought this may be confusing and that's the reason it hasn't been added to the default config before. There is an issue to explore installing it as a plugin: https://github.com/mandiant/VM-Packages/issues/994 Should we wait for that or would you prefer to add the current version now?

Reply:
I think adding it now is fine, since it isn't effecting anything and gives the user the ability to play with it in its current state while also allowing it to be updated in the future if the package changes, without us needing to modify this config again. :)

Reply:
we would need to change the config as the name of the package would change, but is not a big thing. :smile: